server/planning/events/events_spike.py

```diff
@@ -62,6 +62,52 @@ def update(self, id, updates, original):
 
         return item
 
+    def on_updated(self, updates, original):
+        # Spike associated planning
+        planning_spike_service = get_resource_service('planning_spike')
+        for planning in list(get_resource_service('planning').find(where={
```

Comment:
1. It would be better to use `.search(` with a source query to use elastic instead of mongo
2. The `list(` is not required as you're iterating over a cursor instance
3. No need to check if `planning['state'] == 'draft'` as the query will only return draft items